### ISSUE TITLE
Can't pass -- to underlying command

### ISSUE THREAD

**Issue by bchallenor** [mattgreen/watchexec#76]
I need to pass `--` twice, once to `watchexec` and once to the underlying command. But it looks like `watchexec` is losing the second `--`:

```
$ watchexec -r -- echo hello -- world
hello world
^C

$ echo hello -- world
hello -- world
```

Specifically this is important because I want to run `cargo test -- --nocapture`, in which the `--` is mandatory (https://github.com/rust-lang/cargo/issues/296).

**Comment by passcod** [comment]
If you quote the while command you can do that:

    watchexec -r 'cargo test -- --nocapture'

Also works with cargo-watch the same way:

    cargo watch -x 'test -- --nocapture'

Using two `--` without quoting does seem to get lost, I'll look into it.

**Comment by JoshMcguigan** [comment]
I am not seeing this behavior on version `1.92`. Specifically, running `watchexec -r -- echo hello -- world` prints the output `hello -- world`. 

@bchallenor can you confirm you are still seeing this issue?

**Comment by bchallenor** [comment]
Agreed, I do not see this issue any more in 1.9.0. Thanks for the heads up!
